src/progress_bar.rs

```diff
@@ -1,12 +1,17 @@
 use std::borrow::Cow;
-use std::io;
-use std::sync::MutexGuard;
 use std::sync::{Arc, Mutex, Weak};
+use std::sync::{Condvar, MutexGuard};
```

Comment:
Please merge the import statements for `std::sync`.

Reply:
Is it OK if I set up `rustfmt.toml` so this kind of stuff is handled automatically on `cargo fmt`?

Reply:
Sure, sounds great!